### PR TITLE
feat: add serverless and plugins for aws

### PR DESCRIPTION
**Pull request by Shestak2039**
Manual S3 Bucket(403 forbidden because use CloudFront) - http://my-shop.s3-website-eu-west-1.amazonaws.com/
Manual CloudFront - https://d1h5irdf7inxxy.cloudfront.net/
Auto S3 Bucket(403 forbidden because use CloudFront) - http://my-second-shop.s3-website-us-east-1.amazonaws.com/
Auto CloudFront - https://d23gp44fri4ols.cloudfront.net/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,8 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (postc0de) [inline .gitignore:31]: that's redundant, as it is already there by default (line#13)

**Review by postc0de — APPROVED** [review]
Well done!

File: .gitignore (hunk 1 above)
Comment (Shestak2039) [reply .gitignore:31]: Many thanks)

**Review by Salkcid — APPROVED** [review]
Everything's good

**Review by Hantsouski — APPROVED** [review]

**Pull request merged**